rest-api/src/main/scala/pipeline/rest/Main.scala

```diff
@@ -0,0 +1,51 @@
+package pipeline.rest
+
+import scala.concurrent.Await
+import scala.concurrent.Future
+import scala.concurrent.duration.Duration
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.marshallers.xml.ScalaXmlSupport._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.http.scaladsl.model._
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Sink
+import akka.util.ByteString
+import pipeline.data.PipelineCluster
+import scala.collection.JavaConverters._
+import spray.json._
+import pipeline.data._
```

Comment:
do you really need all these imports (e.g., Await and Duration don't seem to be needed)